MycroftAI/mycroft-core
# Add g++ to native dependencies

This fixes #1146. If you run across this PR and are using one of the 3 OSs below, please test and comment if the package installs correctly.

 - [x] Debian / Ubuntu
  - [x] Fedora
- [x] Arch (Untested but verified online)

### Testing instructions

```Bash
git fetch
git checkout bugfix/issue-1146
./dev_setup.sh -sm
```
Now make sure it says something like `Already installed so now going to to be downloaded` rather than something like `Could not find package g++`.
